handlers/main.yml

```diff
@@ -1,7 +1,11 @@
 ---
+- name: supervisord update consul-alerts
+  command: supervisorctl update consul_alerts
+  notify: supervisord restart consul-alerts
+  when: consul_alerts_supervisor_enabled
+
 - name: supervisord restart consul-alerts
   supervisorctl:
-    name: 'consul_alerts:*'
+    name: 'consul_alerts:'
```

Comment:
? Just a question. I think I've read this before and just want to confirm, but those are basically the same for supervisor, correct?